src/_helpers/state-loader.js

```diff
@@ -0,0 +1,36 @@
+class StateLoader {
```

Comment:
This is pretty interesting. I've never messed around with persisting Redux state

Reply:
There was another option to maintain persistence without using local-storage, however that project was depreciated.